### PR TITLE
Add support for helmSecretName in GitRepo 

### PR DESCRIPTION
**Pull request by mantis-toboggan-md**
#2706 
Screen Shot 2021-05-03 at 12 14 24 PM

### CODE REVIEW

File: components/form/SelectOrCreateAuthSecret.vue [hunk 1]
```diff
@@ -256,11 +272,14 @@ export default {
...
           };
         } else {
-          secret._type = SECRET_TYPES.BASIC;
+          secret._type = this.allowCacerts ? SECRET_TYPES.OPAQUE : SECRET_TYPES.BASIC;
           secret.data = {
             username:  base64Encode(this.username),
             password: base64Encode(this.password),
           };
+          if (this.allowCacerts && this.cacerts) {
+            secret.data.cacerts = base64Encode(this.cacerts);
```
Comment (vincent99) [inline components/form/SelectOrCreateAuthSecret.vue:281]: I'm confused why there's a cacert getting tied to the credential secret?  There's already a separate place for that in the gitrepo.  @StrongMonkey ?

Comment (StrongMonkey) [reply components/form/SelectOrCreateAuthSecret.vue:281]: Yeah, I will add that in 2.6. It should be able to reuse that from existing CABundle field.

Comment (vincent99) [reply components/form/SelectOrCreateAuthSecret.vue:281]: Ok so @mantis-toboggan-md you can drop basically all these changes in this file and just have a 2nd select-or-create that's like the first.

Comment (mantis-toboggan-md) [reply components/form/SelectOrCreateAuthSecret.vue:270]: Alright, done

**Pull request merged**